app.py

```diff
@@ -6,6 +6,7 @@
 # Third-party
 import flask
 import talisker.flask
+import talisker.requests
```

Comment:
Why not just `import talisker`?

Reply:
I don't know how talisker imports are configured but it didn't work for me